test/scripts/Dockerfile.tests

```diff
@@ -1,14 +1,16 @@
-FROM node:latest
+FROM node:alpine
+
+RUN apk update && apk add curl
 
 WORKDIR /usr/src/app
-RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.16.0/bin/linux/amd64/kubectl && \
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl && \
 	chmod +x ./kubectl && \
 	mv ./kubectl /usr/local/bin/kubectl
 
 COPY package* ./
 
-RUN npm i
+RUN npm ci
 
 COPY . .
 
-CMD [ "bash", "-c", "--", "while true; do sleep 1; done" ]
+CMD [ "ash", "-c", "while true; do sleep 1; done" ]
```

Comment:
Why the switch to `ash` from `bash`?

Reply:
Because I switched base docker image from `node:latest` to `node:alpine`, so tests take less to run